src/bsconfig.rs

```diff
@@ -157,6 +161,8 @@ pub struct Config {
     pub namespace: Option<NamespaceConfig>,
     pub jsx: Option<JsxSpecs>,
     pub uncurried: Option<bool>,
+    #[serde(rename = "gentypeconfig")]
+    pub gentype_config: Option<GenTypeConfig>,
```

Comment:
I would probably not have this as an empty struct, but rather just `serde::Value` - such that it represents some random piece of JSON which we don't care about. Right now it implies it's empty, rather than something we don't care about.